service/controller/key/appcatalog.go

```diff
@@ -3,8 +3,26 @@ package key
 import (
 	"github.com/giantswarm/apiextensions/v2/pkg/apis/application/v1alpha1"
 	"github.com/giantswarm/microerror"
+
+	"github.com/giantswarm/app-operator/v2/pkg/label"
 )
 
+func CatalogType(customResource v1alpha1.AppCatalog) string {
+	if val, ok := customResource.ObjectMeta.Labels[label.CatalogType]; ok {
+		return val
+	} else {
+		return ""
+	}
+}
+
+func CatalogVisibility(customResource v1alpha1.AppCatalog) string {
+	if val, ok := customResource.ObjectMeta.Labels[label.CatalogVisibility]; ok {
+		return val
+	} else {
```

Comment:
You don't need `else` in here I guess.

Reply:
Thanks this was copy pasta. I fixed it and some other instances in the key package.